todaythediary/debug.log

```diff
@@ -0,0 +1,5 @@
+[1102/165530.511:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Can you ignore these files?

Reply:
@AnandBaburajan did you mean by ignoring the debug.log file? Can you specify more about this, please?

Reply:
> @AnandBaburajan did you mean by ignoring the debug.log file? Can you specify more about this, please?

Aaah, yes. That's what I meant. Remove the file.